docs/modules/ROOT/pages/how-tos/creating.adoc

```diff
@@ -116,6 +116,20 @@ spec:
   url: "https://github.com/konflux-ci/anothertestrepo"
 --
 
+=== Context and Containerfile path
```

Comment:
@arewm Here is the added section in creating.adoc. Please let me know if I should add any edits, I wasn't 100% sure what should go here.

Reply:
I think that I will end up addressing this in my PR, i.e. https://github.com/konflux-ci/docs/pull/126/files#diff-bb9886452ec23549827a5ae772b53b3fbc703dd8e7aa5d196516016cfe8bde46R93-R94

Instead of adding it here, would you be able to comment on my PR to remind me to add a reference where needed?